apps/api/src/campaign-file/dto/files-role.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { Expose } from 'class-transformer'
+import { IsJSON, IsNotEmpty } from 'class-validator'
+
+export class FilesRoleDto {
+  @ApiProperty()
+  @Expose()
+  @IsNotEmpty()
+  @IsJSON({ each: true })
+  filesRole: string
```

Comment:
shouldn't this be an enumeration? I assume the file role cannot be some random text. Last I checked also in the migration it was stated as an enum

Reply:
It's gonna be awesome if we can define the structure better, right now I see the limitation of sending the request as json string instead of proper json object, which is harder to validate properly

Reply:
Can you give me some idea how to validate or how to define structure better?

Reply:
What about the role being an enumeration? I though Pepi was already doing something about this. Please check with him